deployments/docker/agent.yaml

```diff
@@ -0,0 +1,32 @@
+# *Required* The access token for the org that you wish to send metrics to.
+signalFxAccessToken: {"#from": "/etc/signalfx/token"}
+ingestUrl: {"#from": "/etc/signalfx/ingest_url", default: "https://ingest.signalfx.com"}
```

Comment:
Same thing, but the var `$SIGNALFX_INGEST_URL`.

Reply:
Done